### DISCUSSION TITLE
Entry hook cannot take array of parameterized actions (strings)

### DISCUSSION THREAD

**Discussion by OlaoluwaM** [statelyai/xstate#2617] [Q&A]
Hey guys, after bumping to version 4.23.4, I can no longer assign an array of parameterized actions to an entry hook on a state node. Any ideas?

![Screenshot from 2021-09-04 09-58-54](https://user-images.githubusercontent.com/37044906/132089047-b85e7da4-97c3-4988-809d-dd70b744f4b5.png)

**Comment by davidkpiano** [comment]
What does your full machine code look like? E.g., `createMachine<...>( ... )`
